### PR TITLE
fix(ContactVerificationRequestPopup): Fix overlapping messages

### LINKED ISSUE #9002: Text overlapping when verification request question is too long
# Bug Report

Verification request question is limited up to 280 chars and that's not supported by the ui

Screenshot 2023-01-09 at 14 42 23

### PR DESCRIPTION
**Pull request by MishkaRogachev**
Close #9002 and #8950

### What does the PR do

Fix overlapping messages in ContactVerificationRequestPopup

### Affected areas

Profile, ActivityCenter

### Screenshot of functionality (including design for comparison)

Screenshot 2023-01-16 at 15 08 23
Screenshot 2023-01-16 at 14 59 32
Screenshot 2023-01-16 at 14 58 31

### CODE REVIEW

File: ui/imports/shared/popups/ContactVerificationRequestPopup.qml [hunk 1]
```diff
@@ -61,98 +62,85 @@ StatusModal {
...
-        }
+            StatusInput {
+                id: verificationResponse
+                visible: !d.responseText
+                input.multiline: true
+                placeholderText: qsTr("Provide answer to verification request from this contact.")
+                minimumHeight: 152
+                maximumHeight: 152
+                input.verticalAlignment: TextEdit.AlignTop
+                leftPadding: 0
```
Comment (igor-sirotin) [inline ui/imports/shared/popups/ContactVerificationRequestPopup.qml:115]: Are you sure about these zero-paddings?
Looks strange in your screenshot

File: ui/imports/shared/views/chat/SimplifiedMessageView.qml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        StatusSmartIdenticon {
+            id: profileImage
+            name: root.messageDetails.sender.displayName
+            asset: root.messageDetails.sender.profileImage.assetSettings
+            ringSettings: root.messageDetails.sender.profileImage.ringSettings
+
+            MouseArea {
+                cursorShape: enabled ? Qt.PointingHandCursor : Qt.ArrowCursor
+                acceptedButtons: Qt.LeftButton | Qt.RightButton
+                anchors.fill: parent
```
Comment (igor-sirotin) [inline ui/imports/shared/views/chat/SimplifiedMessageView.qml:53]: `anchors.fill: parent` to the top? As per QML styling guidelines 🙂

**Review by igor-sirotin — APPROVED** [review]
Nothing criminal detected 🙂

**Review by caybro — CHANGES_REQUESTED** [review]

**Review by caybro — APPROVED** [review]

**Review by elina2015 — APPROVED** [review]
Tested and approved

**Pull request merged**